packages/flutter_tools/pubspec.yaml

```diff
@@ -48,18 +48,18 @@ dependencies:
   http_multi_server: 3.2.1
   convert: 3.1.1
   async: 2.11.0
-  unified_analytics: 5.0.0
```

Comment:
@christopherfujino @andrewkolos this version update will likely take some time to make it into google3 (git -> gerrit -> google3).. so i'm expecting Google testing to fail for a day or 2

Reply:
Do we have the option to manually roll it? Not necessarily for this change, but to lift the constraint that EACH PR you land in dart-lang/tools, the corresponding tool PR must wait for rolls.

Reply:
I am able to run git -> gerrit by manually opening up a CL pretty easily. This job only runs twice a week so being able to manually do it is ideal.

But for the google3 roll, this job runs daily and I don't know how to do those manually yet

Reply:
Ok, daily is not too bad. Why don't we plan to roll manually to gerrit (btw, where is the gerrit repo?)

Reply:
I do plan on rolling gerrit manually, and the changes are made in the `dart-lang/sdk` repo but it goes through Gerrit when you want to contribute to the sdk, here is the [CL](https://dart-review.googlesource.com/c/sdk/+/333061) for making the version bump

Their [contributing guide](https://github.com/dart-lang/sdk/blob/main/CONTRIBUTING.md#uploading-the-patch-for-review) also explains how to get your CLs submitted to Gerrit

Reply:
oh, if that's the Dart SDK, then that's probably only for the analyzer and dart-dev, right? The copybara script pulls from dart-lang/tools, right (I'm actually asking, I don't really know)? We don't need to manually update the dart sdk unless that's blocking the copybara github -> internal google roll.